src/clientSideScene/sceneEntities.ts

```diff
@@ -1954,14 +1947,6 @@ export class SceneEntities {
         )
       },
       onClick: async (args) => {
-        const firstProfileIndex = Number(sketchNodePaths[0][1][0])
-        const nodePathWithCorrectedIndexForTruncatedAst = structuredClone(
-          mod.pathToNode
-        )
-
-        nodePathWithCorrectedIndexForTruncatedAst[1][0] =
-          Number(nodePathWithCorrectedIndexForTruncatedAst[1][0]) -
-          firstProfileIndex
```

Comment:
there were cases like this, typicall in OnClick handler where it just want's used, but because the variable was modified after declare, lints didn't pick up on it as unused.

Reply:
Nice catch